docs/theme/da_theme_skeleton/layout.html

```diff
@@ -268,7 +260,13 @@
     {% endif %}
   {% endif %}
 
+  {%- if url_root != './' %}
+  <div id="inpageContent"></div>
```

Comment:
This looks strange. Why do we put an empty `div` here? Is some JS going to render into this?

Reply:
No, it's not gonna render. Just copy/pasted the old code that is used to determine if the page is docs.dam.com or not (if it is then don't show the feedback form)

Reply:
What happened to the `<a onclick=...>Detailed Feedback</a>` link?

Reply:
It’s redundant. Previously, we had the blue feedback button that you can see in the screenshot at the bottom right and the detailed feedback button at the top and they both do the same thing. Now that the thumbsup/thumbsdown arrows are directly above the button on the bottom right there is no point in having the separate link.

Reply:
That makes sense. Thanks for the explanation.